drivers/gpu/drm/panel/panel-k6-38-0e-0b-dsc.c

```diff
@@ -18,415 +18,415 @@
-	}
+    ret = mipi_dsi_compression_mode(ctx->dsi, true);
+    if (ret < 0) {
+        dev_err(dev, "failed to enable compression mode: %d\n", ret);
+        return ret;
+    }
 
-	msleep(28); /* TODO: Is this panel-dependent? */
+    msleep(28); /* TODO: Is this panel-dependent? */
 
-	ctx->prepared = true;
-	return 0;
+    ctx->prepared = true;
+    return 0;
 }
 
-static int k6_38_0e_0b_dsc_unprepare(struct drm_panel *panel)
+static int k6_38_0e_0b_dsc_unprepare(struct drm_panel* panel)
 {
-	struct k6_38_0e_0b_dsc *ctx = to_k6_38_0e_0b_dsc(panel);
-	struct device *dev = &ctx->dsi->dev;
-	int ret;
+    struct k6_38_0e_0b_dsc* ctx = to_k6_38_0e_0b_dsc(panel);
+    struct device* dev = &ctx->dsi->dev;
+    int ret;
 
-	if (!ctx->prepared)
-		return 0;
+    if (!ctx->prepared)
+        return 0;
 
-	ret = k6_38_0e_0b_dsc_off(ctx);
-	if (ret < 0)
-		dev_err(dev, "Failed to un-initialize panel: %d\n", ret);
+    ret = k6_38_0e_0b_dsc_off(ctx);
+    if (ret < 0)
+        dev_err(dev, "Failed to un-initialize panel: %d\n", ret);
 
-	gpiod_set_value_cansleep(ctx->reset_gpio, 1);
+    gpiod_set_value_cansleep(ctx->reset_gpio, 1);
 
-	ctx->prepared = false;
-	return 0;
+    ctx->prepared = false;
+    return 0;
 }
 
 static const struct drm_display_mode k6_38_0e_0b_dsc_mode = {
-	.clock = (1080 + 120 + 28 + 120) * (2400 + 20 + 2 + 10) * 120 / 1000,
-	.hdisplay = 1080,
-	.hsync_start = 1080 + 120,
-	.hsync_end = 1080 + 120 + 28,
-	.htotal = 1080 + 120 + 28 + 120,
-	.vdisplay = 2400,
-	.vsync_start = 2400 + 20,
-	.vsync_end = 2400 + 20 + 2,
-	.vtotal = 2400 + 20 + 2 + 10,
-	.width_mm = 69,
-	.height_mm = 154,
+    .clock = (1080 + 120 + 28 + 120) * (2400 + 20 + 2 + 10) * 120 / 1000,
+    .hdisplay = 1080,
+    .hsync_start = 1080 + 120,
+    .hsync_end = 1080 + 120 + 28,
+    .htotal = 1080 + 120 + 28 + 120,
+    .vdisplay = 2400,
+    .vsync_start = 2400 + 20,
+    .vsync_end = 2400 + 20 + 2,
+    .vtotal = 2400 + 20 + 2 + 10,
+    .width_mm = 69,
+    .height_mm = 154,
 };
 
-static int k6_38_0e_0b_dsc_get_modes(struct drm_panel *panel,
-				     struct drm_connector *connector)
+static int k6_38_0e_0b_dsc_get_modes(struct drm_panel* panel,
+    struct drm_connector* connector)
 {
-	struct drm_display_mode *mode;
+    struct drm_display_mode* mode;
 
-	mode = drm_mode_duplicate(connector->dev, &k6_38_0e_0b_dsc_mode);
-	if (!mode)
-		return -ENOMEM;
+    mode = drm_mode_duplicate(connector->dev, &k6_38_0e_0b_dsc_mode);
+    if (!mode)
+        return -ENOMEM;
 
-	drm_mode_set_name(mode);
+    drm_mode_set_name(mode);
 
-	mode->type = DRM_MODE_TYPE_DRIVER | DRM_MODE_TYPE_PREFERRED;
-	connector->display_info.width_mm = mode->width_mm;
-	connector->display_info.height_mm = mode->height_mm;
-	drm_mode_probed_add(connector, mode);
+    mode->type = DRM_MODE_TYPE_DRIVER | DRM_MODE_TYPE_PREFERRED;
+    connector->display_info.width_mm = mode->width_mm;
+    connector->display_info.height_mm = mode->height_mm;
+    drm_mode_probed_add(connector, mode);
```

Comment:
Please switch to drm_connector_helper_get_modes_fixed() in the _get_modes function